Source/WebCore/css/CSSProperties.json

```
@@ -5994,6 +5994,7 @@
                 "uppercase",
                 "lowercase",
                 "full-size-kana",
+                "full-width",
```

```suggestion
                {
                    "value": "full-width",
                    "enable-if": "PLATFORM(COCOA)"
                },
```

Maybe do something like this to allow feature detection to properly work.